clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -9,9 +9,64 @@ import {
   fetchExperience,
 } from "fides-js";
 import { loadPrivacyCenterEnvironment } from "~/app/server-environment";
-import { lookupGeolocation, LOCATION_HEADERS } from "~/common/geolocation";
+import { LOCATION_HEADERS, lookupGeolocation } from "~/common/geolocation";
 
-const FIDES_JS_MAX_AGE_SECONDS = 60 * 60; // one hour
+// one hour, how long the client should cache fides.js for
+const FIDES_JS_MAX_AGE_SECONDS = 60 * 60;
+// one hour, how long until the custom-fides.css is refreshed
+const CUSTOM_FIDES_CSS_TTL_MS = 3600 * 1000;
+
+let cachedCustomFidesCss: string = "";
+let lastFetched: number = 0;
+
+async function fetchCustomFidesCss(
+  req: NextApiRequest
+): Promise<string | null> {
+  const currentTime = Date.now();
+  const shouldRefresh = "refresh" in req.query;
+  if (
+    !cachedCustomFidesCss ||
+    (lastFetched && currentTime - lastFetched > CUSTOM_FIDES_CSS_TTL_MS) ||
+    shouldRefresh
+  ) {
+    try {
+      const environment = await loadPrivacyCenterEnvironment();
+      const fidesUrl =
+        environment.settings.SERVER_SIDE_FIDES_API_URL ||
+        environment.settings.FIDES_API_URL;
+      const response = await fetch(
+        `${fidesUrl}/plus/custom-asset/custom-fides.css`
+      );
+      const data = await response.text();
+
+      console.log("Successfully retrieved custom-fides.css");
```

Comment:
In general all the `console.log` entries are guarded by `process.env.NODE_ENV` to avoid too much log spam (this endpoint is called _a lot_ in production if caches aren't configured in front of it, so the log spam is real)

Reply:
The custom styles are cached so this `console.log` should only happen if the custom styles were retrieved successfully. This happens every hour, or when a manual refresh is triggered.